lib/danger/request_source/github.rb

```diff
@@ -263,13 +262,13 @@ def fetch_danger_repo(organisation: nil)
         data = nil
         data ||= fetch_repository(organisation: organisation, repository: DANGER_REPO_NAME.downcase)
         data ||= fetch_repository(organisation: organisation, repository: DANGER_REPO_NAME.capitalize)
-        return data
+        data
```

Comment:
I actually prefer returns to be more explicit, but I'm fine with the change in this case